pkg/sdk/model/output/kafka.go

```diff
@@ -93,6 +93,16 @@ type KafkaOutputConfig struct {
 	AckTimeout int `json:"ack_timeout,omitempty"`
 	// The codec the producer uses to compress messages (default: nil). The available options are gzip and snappy.
 	CompressionCodec string `json:"compression_codec,omitempty"`
+	// CA certificate
+	SSLCACert *secret.Secret `json:"ssl_ca_cert,omitempty"`
+	// Client certificate
+	SSLClientCert *secret.Secret `json:"ssl_client_cert,omitempty"`
+	// Client certificate chain
+	SSLClientCertChain *secret.Secret `json:"ssl_client_cert_chain,omitempty"`
+	// Client certificate key
+	SSLClientCertKey *secret.Secret `json:"ssl_client_cert_key,omitempty"`
+	// Verify certificate hostname
+	SSLVerifyHostname bool `json:"ssl_verify_hostname,omitempty"`
```

Comment:
I beleive you won't be able to use this flag to actually disable the ssl hostname verification, since the setting behind this is default true. 

Please check the Elasticsearch plugin to see how this scenario could be implemented. (The only downside is that this configuration flag will be added to the config explicitly even if it's value matches the plugin's default)

Reply:
Thanks, I've updated the type to `*bool` which seems to resolve the issue, I've added `ssl_verify_hostname: false` into the test as well.